src/test.rs

```diff
@@ -89,6 +89,37 @@ macro_rules! p {
     };
 }
 
+macro_rules! intf {
```

Comment:
What does `intf` stand for?

Reply:
It's for `Interpreter failing` assertion. There is a similar macro f! for `parser failing` assertion inside `parser.rs` so I thought of doing something similar.